index.bs

```diff
@@ -4690,6 +4691,9 @@ object is modified, those modifications will be seen by anyone
 inspecting the value of the cursor. However modifying such an object
 does not modify the contents of the database.
 
+The <dfn attribute for=IDBCursor>request</dfn> attribute's getter must
+return the [=cursor/request=] of this [=cursor=].
```

Comment:
FWIW, it looks like the spec uses "the cursor" more frequently than "this cursor". (Ideally the spec would be self-consistent already)

Reply:
I copied this from `{{IDBCursor/source}}`, shall I update that too?